### DISCUSSION TITLE
Using Video Clips as a source instead of camera input for testing

### DISCUSSION THREAD

**Discussion by MrTofuuu** [AxisCommunications/discussions#16] [ACAP] [answered]
Hi, I'm trying to set up some automated tests for our application and was curious to see if it was possible to use video clips as a input instead of the camera input. thanks!

**Comment by marbali8** [comment, marked as answer]
Hello!
Thank you for your question. We have been developing this for internal use, but haven't made a product out of it and there are no definite plans to do so at the moment.

**Reply by MrTofuuu** [reply]
Cool! If my company is part of the ACAP program, would we be seeing a possibility for partners to use this?

**Reply by marbali8** [reply]
Not at the moment.
